### PR TITLE
Renames the Wendigo to Hornvandr

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Renames the Wendigo to Hornvandr

Changed all mentions to the latter

Changed dmis, file names, and map names as well

Tested!!! Spawned all hornvrandr things and teh map and it works :)

## Why It's Good For The Game

This has been a dramamama for over a year and i would like to lazily throw my 2 cents into the ring

I am not that stupid I know this isnt a magic fix. But at least it's something until/if nobody changes the theme of this megafauna up.

Before getting mad in the comments read this

image

The 'name cannot be spoken cuz taboo' thing is stupid dumb bullshit and fake, thats not what the problem is about

The problem so far as it is one is a shitty movie director using the name of a genuinely-believed monster in tribal folklore to make a shitty unrelated monster in a movie, which unfortunately created the pop culture depiction of a wendigo

As such there are 2 definitions

1. The Native American one which is just a body-possessing demon spirit

2. The shitty Hollywood one which is some sort of skeleton monster with antlers

3. The secret bonus even shittier Tiktok one which makes up and spreads the lie of Wendigo being a taboo name to even utter (WRONG)

While we can't really magically save the world and remove the second (or third) definition from existence we can at least alter the name to something that actually fits our Hollywood slop monster until someone makes it more unique. Its a pretty lame megafauna anyways

Hornvandr is a portmanteau of 'horn' (horn) and 'vandr' (wanderer) in Old Norse. It sounds cool and thematic too

PROS:
- Hornvandr is a cool name
- No more appropiating culture
- Wendigo was kinda generic and lame anyways
- Name now actually fits the monster as real Wendigos are bodysnatching spirits, and this thing has a horn

CONS:
- Half-assed, lazy fix
- Everyone will still call it a wendigo
- Doesn't really change the design which is clearly a pop culture wendigo

## Changelog

:cl:
spellcheck: Wendigo is now Hornvandr
/:cl:

### CODE REVIEW

File: code/datums/achievements/boss_achievements.dm [hunk 1]
```diff
@@ -103,11 +103,11 @@
...
 	icon_state = "legion"
 
-/datum/award/achievement/boss/wendigo_crusher
-	name = "Wendigo Crusher"
+/datum/award/achievement/boss/hornvandr_crusher
+	name = "Hornvandr Crusher"
 	desc = "You've now ruined years of mythical storytelling."
-	database_id = BOSS_MEDAL_WENDIGO_CRUSHER
-	icon_state = "wendigo"
+	database_id = BOSS_MEDAL_HORNVANDR_CRUSHER
```
Comment (JohnFulpWillard) [inline code/datums/achievements/boss_achievements.dm:109]: Will this currently migrate all wendigo achievements to the new one?

Comment (carlarctg) [reply code/datums/achievements/boss_achievements.dm:109]: i dont know

Comment (Jacquerel) [reply code/datums/achievements/boss_achievements.dm:109]: Don't touch these if you don't know, they're internal vars and don't need to be changed

**Comment by optimumtact** [comment]
I'm not a fan of the fact you didn't do more than a search and replace without thinking through what the impact actually would be.

File: strings/arcade.json [hunk 2]
```diff
@@ -1,164 +1,156 @@
...
-		"Grinch",
-		"Krampus",
-		"Little Drummer Boy",
-		"Scrooge",
-		"Stalking Stocking",
-		"Yeti",
-		"Yuki-Onna"
-	]
-
+  "rpg_adjective": [
```
Comment (optimumtact) [inline strings/arcade.json:2]: can you undo the whitespace changes in this file please.

File: tgstation.dme [hunk 3]
```diff
@@ -3537,7 +3537,7 @@
 #include "code\modules\bitrunning\virtual_domain\domains\syndicate_assault.dm"
 #include "code\modules\bitrunning\virtual_domain\domains\test_only.dm"
 #include "code\modules\bitrunning\virtual_domain\domains\vaporwave.dm"
-#include "code\modules\bitrunning\virtual_domain\domains\wendigo.dm"
+#include "code\modules\bitrunning\virtual_domain\domains\hornvandr.dm"
```
Comment (optimumtact) [inline tgstation.dme:3540]: your DME is out of order now as well.

**Comment by optimumtact** [comment]
Issues aside, i'm okay with this because it's essentially irrelevant, but I think if you're going to change it you may as well just resprite it as well and come up with something entirely new.

File: code/datums/achievements/boss_achievements.dm (hunk 1 above)
Comment (optimumtact) [reply code/datums/achievements/boss_achievements.dm:109]: they will not be migrated automatically so you will need to keep the achievement the same.

**Comment by ZephyrTFA** [comment]
This isnt grammar/formatting but we dont' have a label for "renaming" something that isnt just refactor.
And I'm not giving you refactor gbp.

**Comment by carlarctg** [comment]
> This isnt grammar/formatting but we dont' have a label for "renaming" something that isnt just refactor. And I'm not giving you refactor gbp.

literally 1984

and yeah i didnt know what to label it either

**Comment by the-orange-cow** [comment]
Is it really appropriate to be appropriating words from old Norse culture?

**Comment by LSommers** [comment]
This just seems regressive, honestly.
It's hardly the first time a monster/demon/etc from one culture wound up re-depicted in another culture except fucked up and evil in a different way. I mean shit, look at _just_ European Vampires (because there's a shit-ton of Vampire-like things in basically every culture) and how they went from thirsty off-brand Draugr to plague/pest demons to enthralling and charismatic predators to fucked up and evil charismatic predators, on and on- all this just passing from culture to culture in Europe alone, over time.

There's always some level of cultural bleed, and trying to frame cultural bleed as cultural appropriation is reductive at best. This isn't like some asshat running around with a crude mockery of a ceremonial headdress on because "it looks cool", this is "someone heard a spooky folklore and went 'wow that's spooky', and over time it got twisted and re-adapted into something different but still spooky". 

Also the "modern Wendigo" wasn't make for "Hollywood slop", it was a gradual re-adaption of re-adaptions in various novels since like... 1910, which took the "original Wendigo" and grafted various "decaying nature and animal" sort of elements onto it over time. The look was slowly refined from the original "corrupt, ever-hungry cold corpse" to the non-Ojibwe "fucked up bigfoot with horns" to the deer-skulled monster that is used today.

**Comment by Rex9001** [comment]
I think Bambi is a better name, feels closer to bubblegum. As for Hornvandr its closer to Horn-evil in old norse but vandr sounds close enough to what most people would consider old norse.

**Comment by carlarctg** [comment]
> This just seems regressive, honestly. It's hardly the first time a monster/demon/etc from one culture wound up re-depicted in another culture except fucked up and evil in a different way. I mean shit, look at _just_ European Vampires (because there's a shit-ton of Vampire-like things in basically every culture) and how they went from thirsty off-brand Draugr to plague/pest demons to enthralling and charismatic predators to fucked up and evil charismatic predators, on and on- all this just passing from culture to culture in Europe alone, over time.
> 
> There's always some level of cultural bleed, and trying to frame cultural bleed as cultural appropriation is reductive at best. This isn't like some asshat running around with a crude mockery of a ceremonial headdress on because "it looks cool", this is "someone heard a spooky folklore and went 'wow that's spooky', and over time it got twisted and re-adapted into something different but still spooky".
> 
> Also the "modern Wendigo" wasn't make for "Hollywood slop", it was a gradual re-adaption of re-adaptions in various novels since like... 1910, which took the "original Wendigo" and grafted various "decaying nature and animal" sort of elements onto it over time. The look was slowly refined from the original "corrupt, ever-hungry cold corpse" to the non-Ojibwe "fucked up bigfoot with horns" to the deer-skulled monster that is used today.

thank you for providing new and valuable insight into this 

seeing as the mantainers aggressively dont care about this, the feedback here i agree with, and player malding, i will close the pr as nobody likes it

**Comment by Jacquerel** [comment]
the tactic of "do it but half ass it so that it doesnt happen for another 6 months" employed successfully

**Pull request closed without merging**